PSReadLine/Completion.cs

```diff
@@ -513,6 +515,15 @@ public void DrawMenu(Menu previousMenu, bool menuSelect)
                     }
                 }
 
+                // if the menu has moved, we need to clear the lines under it
+                if (bufferEndPoint.Y < PreviousTop)
+                {
+                    console.BlankRestOfLine();
+                    Singleton.WriteBlankLines(1);
```

Comment:
In an extreme case, we might need multiple blank lines, right? Like `PreviousTop - bufferEndPoint.Y`?

Reply:
It's possible, although highly unlikely.  I can make the change.